prime-router/src/main/kotlin/fhirengine/utils/HL7Reader.kt

```diff
@@ -44,6 +44,10 @@ class HL7Reader(private val actionLogger: ActionLogger) : Logging {
         return messages
     }
 
+    fun isBatch(rawMessage: String, numMessages: Int): Boolean {
+        return rawMessage.startsWith("FHS") || numMessages > 1
+    }
+
     /**
```

Comment:
Functions needs kdoc. It might also be good do add unit tests for this function.